usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -181,11 +179,27 @@ def type_filter(_type, filters, limit=None):
                 award["code"] = code
                 if _type == "award":
                     award["name"] = code
-            actual_total += award["total"]
+            if award["amount"] is None:
+                award["amount"] = 0
+            if award["name"] is None:
+                award["name"] = "Blank {}".format(_type.capitalize().replace("_", " "))
+            actual_total += award["total"] or 0
+
+        result_set = list(alt_set)
+
+        # we need to get the File B data for the same set of filters, so we re-run the spending_filter but without setting the _type to any of the alt keys.
+        alt_set2, queryset2 = spending_filter(alt_set, queryset, filters, "")
+        expected_total = queryset2.aggregate(total=Sum("amount"))["total"]
+        unreported_obj = {"id": None, "code": None, "type": _type, "name": UNREPORTED_FILE_C_NAME, "amount": None}
```

Comment:
I'm not sure these changes are meeting the ticket AC. The ticket mentions two separate cases of missing data which should go into two separate "buckets" for display.
1. Recipient/Award record exists but is unknown/no name
2. Non-Award Spending

Are these two scenarios being rolled into the same result record?